weather-app-orig/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Mini Weather App</title>
+    <link rel="stylesheet" href="css/style.css">
```

Comment:
Make sure you load your custom CSS file after any libraries you are using (i.e. googlefonts, bootstrap, etc.)